### PR TITLE
Translating ch16-00 in French.

### LINKED ISSUE #166: Translating ch16-00
I'm working on the French translation of this page.

Voluntary ignoring #21.

### PR DESCRIPTION
**Pull request by Jimskapt**
Closes #166

### CODE REVIEW

File: FRENCH/src/ch16-00-concurrency.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+-->
+
+Le développement sécurisé et efficace dans des contextes de concurrence est un
+autre objectif majeur de Rust. La *programmation concurrente*, dans laquelle
+différentes parties d'un programme s'exécutent de manière indépendente, et le
+*parallélisme*, dans lequel différentes parties d'un programme s'exécutent en
+même temps, sont devenus des pratiques de plus en plus importantes au fur et à
+mesure que les ordinateurs tirent parti de leurs processeurs multiples.
+Historiquement, le développement dans ces contextes est difficile et favorise
+les erreurs : Rust compte bien changer la donne.
```
Comment (yozhgoor) [inline FRENCH/src/ch16-00-concurrency.md:24]:
```suggestion
Historiquement, le développement dans ces contextes etait difficile et favorisais
les erreurs : Rust compte bien changer la donne.
```

File: FRENCH/src/ch16-00-concurrency.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+De nombreux langages sont dogmatiques sur les solutions qu'ils offrent pour
+gérer les problèmes de concurrence. Par exemple, Erlang a une fonctionnalité
+élégante de passage de messages pour la concurrence mais a une façon étrange
+de partager un état entre les tâches. Ne proposer qu'un sous-ensemble de
+solutions possibles est une stratégie acceptable pour les langages de haut
+niveau, car un langage de haut niveau offre des avantages en sacrifiant
+certains contrôles pour être plus accessibles. Cependant, les langages de bas
+niveau sont censés fournir la solution la plus performante dans n'importe
+quelle situation donnée et proposer moins de facilités sur le matériel.
+Cependant, Rust offre une gamme d'outils répondre aux problèmes de toutes les
```
Comment (yozhgoor) [inline FRENCH/src/ch16-00-concurrency.md:98]:
```suggestion
Cependant, Rust offre une gamme d'outils pour répondre aux problèmes de toutes les
```

File: FRENCH/src/ch16-00-concurrency.md (hunk 1 above)
Comment (Jimskapt) [reply FRENCH/src/ch16-00-concurrency.md:24]:
```suggestion
Historiquement, le développement dans ces contextes était difficile et favorisait
les erreurs : Rust compte bien changer la donne.
```
plutôt, non ?

**Comment by Jimskapt** [comment]
no translation update needed.

**Pull request merged**